prov/efa/src/efa_user_info.c

```diff
@@ -354,6 +355,35 @@ int efa_user_info_alter_rxr(struct fi_info *info, const struct fi_info *hints)
 
 			info->domain_attr->mr_mode |= FI_MR_HMEM;
 
+			support_atomic = true;
+			if (hmem_ops[FI_HMEM_CUDA].initialized == true && cuda_get_xfer_setting() == false) {
```

Comment:
just `hmem_ops[FI_HMEM_CUDA].initialized` do not compare it with true

should be `cuda_xfer_setting() != CUDA_XFER_ENABLED`

cuda_xfer_setting()  is not boolean, it has 3 possible values (0 unspecified, 1 enabled, 2 disabled)

Reply:
done